Descent3/descent.cpp

```diff
@@ -515,7 +505,7 @@ void Descent3() {
           // a lowercase extension
           char extension[16], *p;
           p = extension;
-          ddio_SplitPath(GameArgs[intro_movie_arg + 1], NULL, base_intro_movie_name, extension);
+          ddio_SplitPath(GameArgs[intro_movie_arg + 1], nullptr, base_intro_movie_name, extension);
```

Comment:
You said in description that the "-intro" argument was removed? 8 lines higher we still search for it

Reply:
Yeah, missed that part. Good thing, I was able refactor a little bit intro playback.